config/sync/litteratursiden.settings.yml

```diff
@@ -21,7 +21,7 @@ region_wells:
   footer: ''
 table_responsive: '-1'
 table: ''
-background_path: 'public://stine_pilgaard_topbilledei_0.jpg'
+background_path: 'public://001 Foto- Tale Hendnes 2020-25 (1)_0.jpg'
```

Comment:
Do we override the default background image on release if changed by editors?

Reply:
Have pulled the `prod` db before update, so this is the current setting. But could be argued that we should use config-split or similar to exclude the `background_path`